### PR TITLE
Add blog post about the CZI remote development grant award & job opportunity

### LINKED ISSUE #212: Copyright date on website says 2022
Hello, spyder newbie here.

The copyright date on the website says 2022.

> © 2022 Spyder Website Contributors

Despite knowing better, it still has an effect on my first impression, especially of a project that will need constant maintenance like an IDE (which spyder seems to be getting).

I think changing it to 2023 might improve first impressions a bit.

### PR DESCRIPTION
**Pull request by ccordoba12**
This will allow us to quickly write announcements for the developer we need to hire for this project.

Fixes #212.

### CODE REVIEW

**Comment by CAM-Gerlach** [comment]
I've optimized the screenshot, and updated the copyright date so this also fixes #212 . My other changes will be as review suggestions.

File: content/blog/czi-grant/contents.lr [hunk 1]
```diff
@@ -0,0 +1,37 @@
+title: Improvements to the Spyder IDE installation experience
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
title: Spyder receives CZI grant to add remote development features
```

We probably should update the title :)

File: content/blog/czi-grant/contents.lr [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+---
+author: ccordoba12
+---
+pub_date: 2023-06-08 00:00
+---
+category: Announcement
+---
+tags: grant
+---
+summary: We are pleased to announce that Spyder has received a two-year grant from the Chan Zuckerberg Initiative to add remote development capabilities to our beloved IDE. This means that users will be able to easily take advantage of the vast resources offered by HPC clusters or the cloud to execute their code, while working from the comfort of their personal computers.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]: Great!

File: content/blog/czi-grant/contents.lr [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+---
+body:
+
+During the last few years, Spyder has positioned itself as a popular data science IDE by combining interactive computing and ease of use with robust programming tools.
+However, limited remote development support compared to other IDEs has hindered adoption, as many users would like to work with data and code on high performance computing (HPC) clusters or cloud providers like AWS, GCP or DigitalOcean from their personal computers.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
However, limited remote development support compared to some other IDEs has hindered adoption, as many users would like to work with data and code on high performance computing (HPC) clusters or cloud providers like AWS, GCP or DigitalOcean while developing on their personal computers.
```

* Not all other IDEs offer similar support. 
* Revised the latter phrase to be clearer and flow better

File: content/blog/czi-grant/contents.lr [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+Adding such a feature would open up many new research possibilities by enabling the scientific community to tackle data and compute-intensive programming tasks from the ease and efficiency of their local development environments.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
Adding such features would open up many new research possibilities by enabling the scientific community to tackle data and compute-intensive programming tasks from the ease and efficiency of their local development environments.
```

* Multiple features are involved here, as reflected in other phrasing (capabilities, etc)

File: content/blog/czi-grant/contents.lr [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+And now, thanks to a two-year grant from the Chan Zuckerberg Initiative, we will be able to address this shortcoming.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
Thanks to a two-year grant from the [Chan Zuckerberg Initiative](https://chanzuckerberg.com/), we will be now able to address this shortcoming.
```

* Clean up phrasing and flow
* Link CZI

File: content/blog/czi-grant/contents.lr [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+Right now, users have two options to work remotely when they cannot connect to a server through a web browser or want to use an IDE for its reproducibility and reusuability features.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
Right now, users have two main options to work remotely using a local IDE (aside from a purely web browser-based approach, which is often not available or desirable):
```

I rephrased this to be shorter, clearer, and avoid syntactic ambiguity and complexity

File: content/blog/czi-grant/contents.lr [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+They can either edit and execute their files in a terminal, which is not user-friendly; or start their IDE on the server and display it locally, which is resource-intensive on the remote machine and slow on the local one.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
They can either edit and execute their files in a terminal, which is not user-friendly; or start their IDE on the server and display it locally, which is typically slow, resource-intensive and difficult to set up.
```

Simplify the existing reasoning and add a big additional reason (complexity of setting up e.g. X11 forwarding or RDP is definitely a big blocker in my own and others' experience)

File: content/blog/czi-grant/contents.lr [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+Furthermore, scientists typically find it difficult to sync remote files to the local machine and/or version control their code on the remote one, so remote changes are often lost, out of sync or difficult to integrate with the code developed locally.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
Furthermore, scientists typically find it difficult to sync remote files to the local machine and version control their code on the remote one, so remote changes are often lost, out of sync or difficult to integrate with code developed locally.
```

* Avoid and/or, which is a rather awkward and strongly discouraged construct
* Trim an excess definite article

File: content/blog/czi-grant/contents.lr [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+To address this situation, we will add and enhance support for local Spyder installations to run code and sync files on remote compute servers.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
To address this situation, we will add and enhance support for local Spyder installations to run code and sync files on remote servers and clusters.
```

* Describe remote machines more clearly and comprehensively

File: content/blog/czi-grant/contents.lr [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+The first step in that direction is to enhance the existing remote code execution in Spyder.
+The current workflow for that is error-prone and quite cumbersome.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
The first step in that direction is to enhance the existing remote code execution features in Spyder, as the current workflow is error-prone and quite cumbersome.
```

* Combine sentences to avoid choppy flow
* Add missing word and trim excess ones

File: content/blog/czi-grant/contents.lr [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+It requires users to install the Spyder-Kernels package on the remote machine, manually start a kernel on it and provide the kernel connection file to the local machine.
+
+![Dialog to create a connection to a remote kernel](remote-kernel-connection.png)
+
+Since Spyder does not currently have a way to store or identify state changes, users will have to repeat this costly setup process if there are any configuration modifications on the remote machine.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
Since Spyder does not currently have a way to identify and store state changes, users will have to repeat this costly setup process if there are any configuration modifications on the remote machine.
```

* Flip the order to the natural one

File: content/blog/czi-grant/contents.lr [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+Our plan is to fully automate this procedure so that Spyder can start and connect to a remote kernel automatically, with the user only needing to provide their SSH credentials and some remote environment details.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
Our plan is to fully automate this procedure so that Spyder can start and connect to a remote kernel automatically, with the user only needing to provide their SSH credentials and specify the Python environment they want to use.
```

"Some remote environment details" is rather vauge and could be take to be more labor-intensive/complex than it actually is, so I've clarified it a little

File: content/blog/czi-grant/contents.lr [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+After that is implemented, we will add support to create and manage remote Python environments, explore the remote filesystem in the Files pane, and edit remote files in the Editor.
+This will make most of the core Spyder functionality work to develop code remotely.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
This will make most of the core Spyder functionality work when developing code remotely.
```

* Fix grammar

File: content/blog/czi-grant/contents.lr [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+In addition, users will be able to easily start shells connected to the server after we implement that capability in Spyder-Terminal.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
In addition, users will be able to easily start system shells connected to the server once we implement that capability in Spyder-Terminal.
```

* Clarify the type of shell we're talking about
* Fix a word

File: content/blog/czi-grant/contents.lr [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+This will allow the remote execution of any operating system command, such as Git commit and push instructions.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
This will allow the remote execution of any operating system-level command or program, such as Git commit and push instructions.
```

* Clarify that this isn't just built-in OS commands, but also arbitrary programs.

Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]: This is completely missing any kind of closing or conclusion, it just suddenly trails off. You'll want to add a short paragraph properly wrapping up here.

**Review by CAM-Gerlach — COMMENTED** [review]
A bunch of suggestions, and one comment.

File: content/blog/czi-grant/contents.lr (hunk 6 above)
Comment (ccordoba12) [reply content/blog/czi-grant/contents.lr]: Counterproposal to not sound too dismissive of Jupyter (in case people read this sentence that way)

```suggestion
Right now, users have two main options to work remotely using a local IDE (aside from a purely web browser-based approach, which is sometimes not available or desirable):
```

File: content/blog/czi-grant/contents.lr (hunk 15 above)
Comment (ccordoba12) [reply content/blog/czi-grant/contents.lr]: Good point! I'll add a closing paragraph.

File: content/blog/czi-grant/contents.lr [hunk 16]
```diff
@@ -13,25 +13,30 @@ summary: We are pleased to announce that Spyder has received a two-year grant fr
...
+Since Spyder does not currently have a way to identify and store state changes, users will have to repeat this costly setup process if there are any configuration modifications on the remote machine.
+Our plan is to fully automate this procedure so that Spyder can start and connect to a remote kernel automatically, with the user only needing to provide their SSH credentials and specify the Python environment they want to use.
 
 After that is implemented, we will add support to create and manage remote Python environments, explore the remote filesystem in the Files pane, and edit remote files in the Editor.
-This will make most of the core Spyder functionality work to develop code remotely.
-In addition, users will be able to easily start shells connected to the server after we implement that capability in Spyder-Terminal.
-This will allow the remote execution of any operating system command, such as Git commit and push instructions.
+This will make most of the core Spyder functionality work when developing code remotely.
+In addition, users will be able to easily start system shells connected to the server once we implement that capability in Spyder-Terminal.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
In addition, users will be able to easily start system shells connected to the server once we implement that capability in [Spyder-Terminal](https://github.com/spyder-ide/spyder-terminal).
```

Link Spyder-Terminal

File: content/blog/czi-grant/contents.lr [hunk 17]
```diff
@@ -13,25 +13,30 @@ summary: We are pleased to announce that Spyder has received a two-year grant fr
...
+To address this situation, we will add and enhance support for local Spyder installations to run code and sync files on remote servers and clusters.
+The first step in that direction is to enhance the existing remote code execution features in Spyder, as the current workflow is error-prone and quite cumbersome.
 It requires users to install the Spyder-Kernels package on the remote machine, manually start a kernel on it and provide the kernel connection file to the local machine.
 
 ![Dialog to create a connection to a remote kernel](remote-kernel-connection.png)
 
-Since Spyder does not currently have a way to store or identify state changes, users will have to repeat this costly setup process if there are any configuration modifications on the remote machine.
-Our plan is to fully automate this procedure so that Spyder can start and connect to a remote kernel automatically, with the user only needing to provide their SSH credentials and some remote environment details.
+Since Spyder does not currently have a way to identify and store state changes, users will have to repeat this costly setup process if there are any configuration modifications on the remote machine.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
Since Spyder does not currently have a way to identify and store state changes, users currently have to repeat this costly setup process if there are any configuration modifications on the remote machine.
```

Emphasize this is the current state

File: content/blog/czi-grant/contents.lr [hunk 18]
```diff
@@ -13,25 +13,30 @@ summary: We are pleased to announce that Spyder has received a two-year grant fr
...
+Our plan is to fully automate this procedure so that Spyder can start and connect to a remote kernel automatically, with the user only needing to provide their SSH credentials and specify the Python environment they want to use.
 
 After that is implemented, we will add support to create and manage remote Python environments, explore the remote filesystem in the Files pane, and edit remote files in the Editor.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
After that is implemented, we will add support to create and manage remote Python environments, explore the remote filesystem in the [Files pane](https://docs.spyder-ide.org/current/panes/fileexplorer.html), and edit remote files in the [Editor](https://docs.spyder-ide.org/current/panes/editor.html).
```

May as well cross-link these too

File: content/blog/czi-grant/contents.lr [hunk 19]
```diff
@@ -13,25 +13,30 @@ summary: We are pleased to announce that Spyder has received a two-year grant fr
...
-This will make most of the core Spyder functionality work to develop code remotely.
-In addition, users will be able to easily start shells connected to the server after we implement that capability in Spyder-Terminal.
-This will allow the remote execution of any operating system command, such as Git commit and push instructions.
+This will make most of the core Spyder functionality work when developing code remotely.
+In addition, users will be able to easily start system shells connected to the server once we implement that capability in Spyder-Terminal.
+This will allow the remote execution of any operating system-level command or program, such as Git commit and push instructions.
+
+We hope you will be as thrilled as we are by this fantastic news!
+Another interesting announcement for the Scientific Python community is that we are hiring for this project.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
Finally, we want to announce that we're hiring for this project!
```

* Use much more appropriate and direct phrasing
* The scientific Python here is not the (proper noun) "Scientific Python" organization, but rather the (common noun) "scientific Python" community.

File: content/blog/czi-grant/contents.lr [hunk 20]
```diff
@@ -13,25 +13,30 @@ summary: We are pleased to announce that Spyder has received a two-year grant fr
...
+We are looking for a developer with solid networking knowledge in Python to help us implement the backend infrastructure that will power the features described above.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
We are looking for a Python developer with solid networking knowledge to help us implement the backend infrastructure that will power the features described above.
```

Say this a little more cleanly

File: content/blog/czi-grant/contents.lr [hunk 21]
```diff
@@ -13,25 +13,30 @@ summary: We are pleased to announce that Spyder has received a two-year grant fr
...
+This is a part time job to work with NumFOCUS as a contractor, it does not require experience with PyQt (although some would be beneficial) and it lasts until the end of the year, with the possibility to renew the contract the next one.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
This is a part time position to work as a contractor through [NumFOCUS](https://numfocus.org/); it does not require experience with PyQt (although some would be beneficial) and it lasts until the end of the year, with the possibility to renew the contract for the next.
```

* Re-arrange phrasing for clearer and cleaner flow
* Fix grammar error (comma splice)
* Link NumFOCUS
* Remove redundant word

File: content/blog/czi-grant/contents.lr [hunk 22]
```diff
@@ -13,25 +13,30 @@ summary: We are pleased to announce that Spyder has received a two-year grant fr
...
+Interested people can [contact me](ccordoba12@gmail), to talk about it.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
Interested?
[Contact me](mailto:ccordoba12@gmail.com) to talk more about it!

We hope you will be as thrilled as we are by this fantastic news, and are looking forward to sharing more with you on this project in the near future!
```

* Adopt a more personal and compelling tone
* Fix broken mailto link (that is actually a local 404, since its missing mailto)
* Add a missing word
* Add the previous first sentence of this para, that didn't fit with the rest, as a proper closing instead

File: content/blog/czi-grant/contents.lr [hunk 23]
```diff
@@ -13,25 +13,30 @@ summary: We are pleased to announce that Spyder has received a two-year grant fr
...
 
 After that is implemented, we will add support to create and manage remote Python environments, explore the remote filesystem in the Files pane, and edit remote files in the Editor.
-This will make most of the core Spyder functionality work to develop code remotely.
-In addition, users will be able to easily start shells connected to the server after we implement that capability in Spyder-Terminal.
-This will allow the remote execution of any operating system command, such as Git commit and push instructions.
+This will make most of the core Spyder functionality work when developing code remotely.
+In addition, users will be able to easily start system shells connected to the server once we implement that capability in Spyder-Terminal.
+This will allow the remote execution of any operating system-level command or program, such as Git commit and push instructions.
+
+We hope you will be as thrilled as we are by this fantastic news!
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
```

Move this to be the final closing, as it fits better there and avoid burying the lede of the job offer

**Review by CAM-Gerlach — COMMENTED** [review]
Some comments on the new concluding paragraph, and a few other things I spotted in the rest of the text.

File: content/blog/czi-grant/contents.lr (hunk 15 above)
Comment (CAM-Gerlach) [reply content/blog/czi-grant/contents.lr]: Thanks!

File: content/blog/czi-grant/contents.lr (hunk 6 above)
Comment (CAM-Gerlach) [reply content/blog/czi-grant/contents.lr]: SGTM :+1: I left it so you can batch-apply it with the other suggestions

File: content/blog/czi-grant/contents.lr [hunk 24]
```diff
@@ -0,0 +1,41 @@
+title: Spyder receives CZI grant to add remote development features
```
Comment (ccordoba12) [inline content/blog/czi-grant/contents.lr]:
```suggestion
title: Spyder receives CZI grant to add remote development features and job opportunity thanks to it
```

File: content/blog/czi-grant/contents.lr (hunk 22 above)
Comment (ccordoba12) [reply content/blog/czi-grant/contents.lr]: I think `forward` is missing after `looking` in your suggestion?

**Review by ccordoba12 — COMMENTED** [review]
Last comments, then this should be ready for me.

File: content/blog/czi-grant/contents.lr (hunk 22 above)
Comment (CAM-Gerlach) [reply content/blog/czi-grant/contents.lr]: Yup, it also needed a couple other tweaks. Updated, thanks.

File: content/blog/czi-grant/contents.lr (hunk 24 above)
Comment (CAM-Gerlach) [reply content/blog/czi-grant/contents.lr]: Hmm, waaaaay too wordy. What about this?

```suggestion
title: Spyder gets a CZI grant to add remote development features, and a new job opening!
```

File: content/blog/czi-grant/contents.lr [hunk 25]
```diff
@@ -0,0 +1,41 @@
...
+---
+author: ccordoba12
+---
+pub_date: 2023-06-08 00:00
+---
+category: Announcement
+---
+tags: grant
+---
+summary: We are pleased to announce that Spyder has received a two-year grant from the Chan Zuckerberg Initiative to add remote development capabilities to our beloved IDE. This means that users will be able to easily take advantage of the vast resources offered by HPC clusters or the cloud to execute their code, while working from the comfort of their personal computers.
```
Comment (CAM-Gerlach) [inline content/blog/czi-grant/contents.lr]:
```suggestion
summary: Spyder has received a two-year grant from the Chan Zuckerberg Initiative to add remote development capabilities to our beloved IDE. This means you'll be able to easily take advantage of the vast resources offered by HPC clusters or the cloud to execute your code, while working from the comfort of your personal computer. And if you're interested in helping us implement that, we may have just the job opportunity for you!
```

* Add mention of job opportunity here, too
* Reframe in more engaging, personal style ("you")

**Review by CAM-Gerlach — APPROVED** [review]
LGTM now, thanks @ccordoba12
